### ISSUE TITLE
Way to describe CNV gain of greater than a dup (ie ISCN x4)

### ISSUE THREAD

**Issue by davmlaw** [HGVSnomenclature/hgvs-nomenclature#52]
International System for Human Cytogenomic Nomenclature (ISCN) has "x1", "x3", "x4" etc

x1 is a HGVS del
x3 is a HGVS dup
x4 on different chromosomes could be "homozygous for a dup" but if it is x1 on one chromosome and x3 on the other there is no easy way to describe it in HGVS (if it is very large)

There are 1093 entries for x4 in ClinVar - example - https://www.ncbi.nlm.nih.gov/clinvar/variation/687667/

GRCh37/hg19 7q11.21(chr7:63903253-64426719)x4

There is [repeated sequences](https://varnomen.hgvs.org/recommendations/DNA/variant/repeated/) but that appears to be for when the repeat is small, ie not "del/dup" sized

**Suggestion for new nomenclature**

You could potentially argue for just adding x4 on the end (inkeeping with ISCN) however if you wanted to style it like a mashup of repeat sequences + del/dup perhaps you could describe:

GRCh37/hg19 7q11.21(chr7:63903253-64426719)x4 

in HGVS as:

NC_000007.13:g.63903253_64426719mul[4]

So these are equivalent:

NC_000007.13:g.63903253_64426719del = NC_000007.13:g.63903253_64426719mul[1]
NC_000007.13:g.63903253_64426719dup = NC_000007.13:g.63903253_64426719mul[3]

**Comment by Peter-J-Freeman** [comment]
A thing to consider here is that the HGVS nomenclature reserved "dup" for tandemly duplicated sequence, unlike VCF which does not apply this restriction.

I like the idea of 

```
NC_000007.13:g.63903253_64426719del = NC_000007.13:g.63903253_64426719cnv[1]
NC_000007.13:g.63903253_64426719dup = NC_000007.13:g.63903253_64426719cnv[3]
``

but this format suggests we know the exact number of copies of the duplicated region, and may break if we do not. So how about something like

NC_000007.13:g.63903253_64426719del = NC_000007.13:g.63903253_64426719cnl[1]
NC_000007.13:g.63903253_64426719dup = NC_000007.13:g.63903253_64426719cng[3]

where cnl states an overall loss of copies and cng an overall gain of copies and the base format can be 

NC_000007.13:g.63903253_64426719del = NC_000007.13:g.63903253_64426719cnl
NC_000007.13:g.63903253_64426719dup = NC_000007.13:g.63903253_64426719cng 

if the exact number of copies is not known.

**Comment by davmlaw** [comment]
Instead of thr number being copies, you could store the change, ie:

```
100_200cnv[-1] == :g.100_200del
:g.100_200cnv[+1] == :g.100_200dup
```

**Comment by jfjlaros** [comment]
> NC_000007.13:g.63903253_64426719dup = NC_000007.13:g.63903253_64426719cnv[3]
[...]
> 1:g.100_200cnv[+1] == 1:g.100_200dup

A duplication is not equivalent to a copy number gain, i.e., every duplication can be written down as a CNV (losing information about the location of the duplicated region), but not the other way around.

With regards to absolute or relative counts, perhaps we can also use a couple of examples from chrY to see what feels more natural.

**Comment by Peter-J-Freeman** [comment]
Agreed, the cnv description must be distinct from the dup description.

**Comment by jtdendunnen** [comment]
regarding this discussion, we already have [2] to indicate two copies of a sequence which is distinct from dup for a duplication. Why we need "cnv" is not clear to me.

**Comment by Peter-J-Freeman** [comment]
@jtdendunnen. Please provide the links to the varnomen page if possible and an example of use. 

My argument would be that HGVS reserved the dup syntax for a tandem duplication so we should not use that. Also, we ideally need a way to state a copy number change and whether the copy number is increased or decreased. It could be we already have rules for this that could be used but the relevant place in the website and a couple of examples would be useful.

**Comment by davmlaw** [comment]
@jtdendunnen  I think you mean [repeated sequences](https://varnomen.hgvs.org/recommendations/DNA/variant/repeated/) e.g. ```NC_000003.12:g.63912687AGC[13]``` 

As far as I can tell from the spec they must have a sequence, which means they are good for small repeats but not for large CNVs

**Comment by jtdendunnen** [comment]
when I have sequenced the break point I should now where an additional copy is located and how to describe it. When I did not sequence the break point and when I am not sure whether the extra copy is in tandem (so a duplication) or not (so an insertion) I describe the variant like RefSeq:g.(A_B)_(C_D)[2] where A/D gives the max. and B/C the min. extent of the extra copy (described as [2]).

**Comment by davmlaw** [comment]
Yeah, I agree that we should distinguish between repeating in place (+1 extra copy = dup, +2 being a tripling of a sequence) vs being an insertion

So maybe 'cnv' isn't a good descriptor term as that can be untethered from location (eg a x3 could be dup plus an ins elsewhere) and really just means amplification

So maybe 'rpt'?

**Comment by davmlaw** [comment]
I don't know if it's a good idea to allow HGVS to describe copying to a new or unknown location...

Though if you can, it would allow you to describe Trisomy 21 in HGVS  as:

NC_000021.9:g.1_46709983cnv[+1]

**Comment by Peter-J-Freeman** [comment]
This also would bring HGVS in line with VCF and VRS which can describe cnv increase or decrease agnostic of knowing the new location.

**Comment by jtdendunnen** [comment]
> 
Trisomy 21 in HGVS is described as NC_000021.9:g.pter_qtersup  (sup for supernumary chromosome)

**Comment by ifokkema** [comment]
I think part of this conversation is based on the thought that the variant description is for the entire organism. E.g;

> So these are equivalent:
> 
> NC_000007.13:g.63903253_64426719del = NC_000007.13:g.63903253_64426719cnv[1] NC_000007.13:g.63903253_64426719dup = NC_000007.13:g.63903253_64426719cnv[3]

This is not correct; an HGVS variant description is based on one molecule, one allele. I.e., the proposed `NC_000007.13:g.63903253_64426719cnv[3]` would state that there are **four** copies of this region in the individual. Three copies on one allele, additionally to the normal copy on the other allele. I.e., `NC_000007.13:g.63903253_64426719cnv[3]` == `NC_000007.13:g.[63903253_64426719[3]];[63903253_64426719[1]]`.
See here for more information:
https://hgvs-nomenclature.org/en/latest/recommendations/DNA/alleles/

So any variant description coming out of this discussion should always be allele-based, as HGVS is allele-based.
- The proposed `g.100_200cnv[-1]` would equal a simple deletion and should therefore be written as `g.100_200del`. The other allele, if present, is assumed normal if not mentioned.
- `g.100_200cnv[-2]` is not possible as those positions are only present once in that reference sequence. This should, therefore, be written as `g.[100_200del];[100_200del]`.
- `g.100_200cnv[+1]` or `g.100_200[+1]` makes just as much sense as `g.100_200cnv[2]` or `g.100_200[2]` in my opinion.
- `g.100_200cnv[+2]` could be `g.100_200cnv[3]` (`g.100_200[3]`), or `g.[100_200cnv[2]];[100_200cnv[2]]` (`g.[100_200[2]];[100_200[2]]`), or in theory, even a +3 combined with a deletion. I don't remember from the top of my head whether there is a rule that indicates you're not sure which event took place.
- Note that, although the syntax using `[2]` is currently in use, it's not endorsed by the HVNC, and therefore, isn't official HGVS nomenclature.

**Comment by jfjlaros** [comment]
> HGVS can't currently describe copying to a new or unknown location...

The insertion syntax can be used to describe a copy to a new location, e.g.,

[`NC_000021.9:g.10000000_10000001ins20000000_20000030`](https://mutalyzer.nl/normalizer/NC_000021.9:g.10000000_10000001ins20000000_20000030)

or when the inserted sequence originates from a different chromosome:

[`NC_000021.9:g.10000000_10000001insNC_000022.7:g.20000000_20000030`](https://mutalyzer.nl/normalizer/NC_000021.9:g.10000000_10000001insNC_000022.7:g.20000000_20000030)

Copying to an unknown location cannot be expressed as far as I know.

**Comment by jtdendunnen** [comment]
HGVS does have a format for an insertion at another position in the genome, see http://varnomen.hgvs.org/recommendations/uncertain/. Correct is NC_000021.9:g.10000000_10000001ins[NC_000022.7:g.20000000_20000030]

**Comment by ifokkema** [comment]
> Copying to an unknown location cannot be expressed as far as I know.

The [nomenclature guidelines](https://hgvs-nomenclature.org/en/latest/recommendations/DNA/insertion/) provide this example:

> `g.?_?ins[NC_000023.10:g.(12345_23456)_(34567_45678)]`: the insertion of a sequence from the X-chromosome (NC_000023.10), maximally involving nucleotides 12345_45678 but certainly nucleotides 23456_34567, at an unknown position (g.?_?) in the genome

Two more pages indicate similar examples. However, the major issue here is that a reference sequence is required, meaning at least the chromosome in which the insertion took place should be provided. If the chromosome in which the insertion took place is unknown, no reference sequence identifier can be added to the variant description, and it'll be impossible to generate a valid HGVS-compliant variant description.
